src/plugins/controls/public/options_list/components/options_list_popover_suggestions.tsx

```diff
@@ -145,6 +149,19 @@ export const OptionsListPopoverSuggestions = ({
     }
   }, [loadMoreSuggestions, totalCardinality]);
 
+  const renderOption = useCallback(
+    (option, searchStringValue) => {
+      if (!allowExpensiveQueries || searchTechnique === 'exact') return option.label;
```

Comment:
There is no reason to highlight the search term for exact match searching - otherwise, you can get some slightly odd behaviour where "matching" substrings get highlighted before the suggestions actually fully load:

![Dec-05-2023 12-20-17](https://github.com/elastic/kibana/assets/8698078/d4db7c50-0b91-4850-8914-12d3aeea3e93)